### PR TITLE
ESLint setup and code alterations

### PR DESCRIPTION
**Pull request by diversifiedmechanic**
After getting ESLint set up, I had to do some code refactoring to get it within the standards.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "env": {
+      "browser": true,
+      "node": true
+  },
+  "extends": "airbnb",
+  "ignorePatterns": ["bundle.js"],
+  "rules": {
+    "no-console": "off"
+  }
+}
```
Comment (ZacharyAb) [inline .eslintrc.js]: lgtm

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 /node_modules
 /.env
+/public/bundle.js
```
Comment (ZacharyAb) [inline .gitignore]: Good ignore here

File: database/index.js [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+db.once('open', () => console.log('Connected to the db'));
 
 // schema for item
 const Schemas = require('./Schemas.js');
 
 
 module.exports.addNewItem = (newItem) => {
-
-  let itemToBeAdded = new Schemas.Item({
+  const itemToBeAdded = new Schemas.Item({
```
Comment (ZacharyAb) [inline database/index.js]: Seems like a good change

File: database/index.js [hunk 4]
```diff
@@ -33,20 +33,20 @@ module.exports.addNewItem = (newItem) => {
...
+    ))
+    .then((allSellerItems) => {
+      // removing the original item from the returned array
+      const filteredSellerItems = allSellerItems.filter((item) => (
+        item.itemId !== Number(idOfItem)
+      ));
+      return filteredSellerItems;
+    })
+);
+
```
Comment (ZacharyAb) [inline database/index.js]: This changes makes a lot of sense to me, great to see the use of ES6 syntax in action

File: dummyData/dataSeeder.js [hunk 5]
```diff
@@ -28,41 +26,41 @@ const databaseSeeder = (numOfSellers = 10, numOfItemsPerSeller = 10) => {
...
+    while (j < numOfItemsPerSeller + 10) {
+      const generatedItemInfo = {};
 
       // makes a copy of the seller as to not reference the same obj in memory
-      for (let key in generatedSellerInfo) {
-        generatedItemInfo[key] = generatedSellerInfo[key];
-      }
+      Object.entries(generatedSellerInfo).forEach(([key, value]) => {
+        generatedItemInfo[key] = value;
+      });
```
Comment (ZacharyAb) [inline dummyData/dataSeeder.js]: Good switch here, I'll be excited to see it pay off in the rendering of the page.

File: dummyData/dataSeeder.js [hunk 6]
```diff
@@ -28,41 +26,41 @@ const databaseSeeder = (numOfSellers = 10, numOfItemsPerSeller = 10) => {
...
-    }).
-    catch((error) => {
+    })
+    .catch((error) => {
       // console logs the error then excess the node process
       console.error('Seeding the data had the following error:', error);
       process.exit();
-    })
+    });
 };
```
Comment (ZacharyAb) [inline dummyData/dataSeeder.js]: Good syntax catches.

File: server/index.js [hunk 7]
```diff
@@ -7,15 +7,15 @@ const app = express();
...
+  db.getAllSellerItemsExceptCurrentItem(req.params.id)
+    .then((allItems) => {
       res.status(200).json(allItems);
-    }).
-    catch((error) => {
+    })
+    .catch((error) => {
       res.status(500).json(error);
     });
 });
```
Comment (ZacharyAb) [inline server/index.js]: Good formatting here!

File: dummyData/dataSeeder.js (hunk 5 above)
Comment (diversifiedmechanic) [reply dummyData/dataSeeder.js]: Me too.  ESLint made me do a lot of strange stuff but it all makes sense.  Hopefully it pays off.

**Pull request merged**